guide/14-deep-learning/how-named-entity-recognition-works.ipynb

```diff
@@ -16,24 +16,38 @@
     "<ul class=\"toc-item\">\n",
```

Comment:
<blockquote><strong>F1</strong>&nbsp;- F1 can be interpreted as a weighted harmonic mean of the precision and recall</blockquote><blockquote><br></blockquote><p>full stop missing</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/799/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
fixed